clientlib-generator.js

```diff
@@ -0,0 +1,46 @@
+var clientlib = require("aem-clientlib-generator");
+var path = require("path");
+var process = require("process");
+
+
+const CLIENTLIB_DIR = process.env.npm_config_directory || "libs/fd/af/themes/corecomponents/canvas3-theme/clientlibs/"
```

Comment:
/libs path is an internal path and should not be exposed in open source.

Reply:
Removed this path